### PR TITLE
Add mAP analysis

### PR DESCRIPTION
**Pull request by MikeLippincott**
This PR adds the mAP visualization to one of the main figures

### CODE REVIEW

File: figures/4.figure4/figures/figure4.png
Comment (MattsonCam) [inline figures/4.figure4/figures/figure4.png:1]: Consider spacing out the x-axis labels in D

Comment (MattsonCam) [reply figures/4.figure4/figures/figure4.png:1]: Also, don't entirely understand this correlation plot

**Review by MattsonCam — APPROVED** [review]
LGTM @MikeLippincott, good job!

File: figures/4.figure4/figures/figure4.png
Comment (MikeLippincott) [reply figures/4.figure4/figures/figure4.png:1]: We can talk about it in person on Monday! I addressed the comment about x-axis spacing - added!

**Pull request merged**